Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/device/BackgroundModeStatus.kt

```diff
@@ -90,7 +90,10 @@ class BackgroundModeStatus @Inject constructor(
         .onStart { emit(pollisIgnoringBatteryOptimizations()) }
         .distinctUntilChanged()
         .onCompletion {
-            if (it != null) Timber.w(it, "isIgnoringBatteryOptimizations failed.")
+            when {
+                it is CancellationException -> Timber.d("isIgnoringBatteryOptimizations canceled.")
+                it != null -> Timber.w(it, "isIgnoringBatteryOptimizations failed.")
```

Comment:
is `it`  an error ? If so could you please use `Timber.e`

Reply:
`it: Throwable?`

I'll adjust the log level to `e`